tests/trace/test_weave_client.py

```diff
@@ -1499,3 +1499,25 @@ def test_object_version_read(client):
                 digest="v1",
             )
         )
+
+
+@pytest.mark.asyncio
```

Comment:
I think the main thing to point out here is that this style of call display naming is safer for the multithreaded case where setting `my_op.call_display_name = ...` might be unsafe.

Maybe the docs should reflect that too (and maybe we should discourage the former?)

Reply:
well it is fundamentally a different use case - rather than naming all calls, we are naming just the one call

Reply:
hmm, ok fair